test/known_issues/test-vm-inherited_properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+// Ref: https://github.com/nodejs/node/issues/5350
```

Comment:
Yea. I meant the known issue tests (I **think** all of them) include `Refs:` at the top.